web/src/message_view.ts

```diff
@@ -79,7 +88,7 @@ export function reset_ui_state(opts) {
     compose_banner.clear_message_sent_banners(true, skip_automatic_new_visibility_policy_banner);
 }
 
-export function changehash(newhash, trigger) {
+export function changehash(newhash: string, trigger?: string): void {
```

Comment:
I think `trigger` is intended to be a mandatory parameter, so we should make it so in a follow-up...

Reply:
Oops, yup it's always required, my bad. Thanks!